### PR TITLE
Change default overlay position to UNDER_WIDGETS

### LINKED ISSUE #451: UI Overlays are drawn over game widgets
Happens in many instances.

Enemy health bar covering NMZ Absorption potion overlay

Attack style indicator covering bank interface

### PR DESCRIPTION
**Pull request by deathbeam**
In order to not draw UI overlays over bank interfaces (and similar popup
interfaces), change default overlay position to UNDER_WIDGETS.

Fixes #451

Signed-off-by: Tomas Slusny <slusnucky@gmail.com>

Preview:
screenie

screenie2

screenie

### CODE REVIEW

**Comment by deathbeam** [comment]
Tested the patch, the current (and only) problems:

* Health bars and overhead prayers rendering over player names (present in master as well)
* Health bars and overhead prayers rendering over some UI elements (happening in rare cases, as mostly they do not intercept with the UI elements in corners)
* In-game widgets rendering over the UI (like pest control void knight HP indicator) - In master, there is different problem with this, and that is UI elements drawing over the in game widgets like this, so this is separate issue

**Comment by deathbeam** [comment]
Fixed all the issues mentioned in the previous comment, here are some examples that it works now:

screenie2

screenie

File: runelite-client/src/main/java/net/runelite/client/ui/overlay/OverlayLayer.java [hunk 1]
```diff
@@ -27,17 +27,22 @@
 public enum OverlayLayer
 {
 	/**
-	 * Render overlay above all game elements
+	 * Render right above surface with actors and objects
 	 */
-	ALWAYS_ON_TOP,
+	ABOVE_SURFACE,
```
Comment (Adam-) [inline runelite-client/src/main/java/net/runelite/client/ui/overlay/OverlayLayer.java]: The consensus seems to be to rename this to ABOVE_SCENE

Comment (deathbeam) [reply runelite-client/src/main/java/net/runelite/client/ui/overlay/OverlayLayer.java]: Done and updated

**Pull request merged**